File: LeetCode/C#/2037. Minimum Number of Moves to Seat Everyone.cs
public class Solution {
    public int MinMovesToSeat(int[] seats, int[] students) {
        var minMoves = 0;
        
        Array.Sort(seats);
        Array.Sort(students);
        
        for(var i = 0; i < seats.Length; i++)
        {
            minMoves += Math.Abs(seats[i] - students[i]);
        }
        
        return minMoves;
    }
}

File: LeetCode/C++/2037. Minimum Number of Moves to Seat Everyone.cpp
class Solution
{
public:
    int minMovesToSeat(vector<int> &seats, vector<int> &students)
    {

        int count = 0;
        sort(seats.begin(), seats.end());
        sort(students.begin(), students.end());

        for (int i = 0; i < seats.size(); i++)
        {
            count += abs(seats[i] - students[i]);
        }

        return count;
    }
};